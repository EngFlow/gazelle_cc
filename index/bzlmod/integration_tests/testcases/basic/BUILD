# gazelle:cc_group unit
# gazelle:cc_indexfile generated.ccindex
