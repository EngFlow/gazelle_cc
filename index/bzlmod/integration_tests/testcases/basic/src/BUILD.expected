load("@rules_cc//cc:defs.bzl", "cc_binary")

cc_binary(
    name = "fmt",
    srcs = ["fmt.cpp"],
    deps = ["@fmt"],
)

cc_binary(
    name = "benchmark",
    srcs = ["benchmark.cc"],
    deps = ["@google_benchmark//:benchmark_main"],
)

cc_binary(
    name = "zlib",
    srcs = ["zlib.cc"],
    deps = ["@zlib"],
)
